src/main/java/mate/academy/spring/dto/response/MovieResponseDto.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.spring.dto.response;
 
 public class MovieResponseDto {
+
     private Long movieId;
+
```

Comment:
do not add empty lines **to all** ResponseDtos

Reply:
fixed